src/@narative/gatsby-theme-novela/components/Logo/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from '@components/Logo/Logo'
```

Comment:
I'll help you to set up a git pre-commit hook later to avoid this kind of problem happening in the future.
![image](https://user-images.githubusercontent.com/6315466/65822950-f8d15f00-e276-11e9-86dd-0e95a6b68bd7.png)